R/pkg/R/mllib_regression.R

```diff
@@ -100,6 +120,12 @@ setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
               print(family)
               stop("'family' not recognized")
             }
+            # Handle when family = statmod::tweedie()
+            if (tolower(family$family) == "tweedie" && !is.null(family$variance)) {
```

Comment:
i assume it handles the "fake" family created on L111 correctly? it doesn't have `variance`